webpack/JobWizard/steps/AdvancedFields/__tests__/AdvancedFields.test.js

```diff
@@ -4,14 +4,15 @@ import configureMockStore from 'redux-mock-store';
 import * as patternfly from '@patternfly/react-core';
 import { mount } from '@theforeman/test';
 import { AdvancedFields } from '../AdvancedFields';
+import { jobTemplateResponse } from '../../../__tests__/fixtures';
 
 jest.spyOn(patternfly, 'FormGroup');
 patternfly.FormGroup.mockImplementation(props => (
   <div>{props.navAriaLabel}</div>
 ));
 const mockStore = configureMockStore([]);
 const store = mockStore({
-  JOB_TEMPLATE: { response: { effective_user: { overridable: true } } },
+  JOB_TEMPLATE: { response: jobTemplateResponse },
```

Comment:
If you want, you can remove this snapshot test file so you don't need to maintain it in other PRs as well, in continuous to the discussion in https://community.theforeman.org/t/rethinking-react-redux-folder-structure/24183/8

you can move the integration test into this file

Reply:
Will move that in a separate pr, thanks!